variables.tf

```diff
@@ -194,6 +210,17 @@ variable "public_access_master_instance" {
   default     = false
 }
 
+variable "allocated_ip_range" {
+  description = <<-EOT
+  The name of the allocated ip range for the private ip CloudSQL instance. 
+  For example: "google-managed-services-default". If set, the instance ip will be created in the allocated range. 
+  The range name must comply with RFC 1035. Specifically, the name must be 1-63 characters long and match the regular expression a-z?. 
+  If set, a CIDR range of /20 is advised.
```

Comment:
The description is saying to provide the name of an allocated IP range which will get used by CloudSQL. Where is that name defined in to begin with? From where can we obtain that name? We should clarify that in the description.

Reply:
done